### ISSUE TITLE
Admin: unmerge/break up all events on a player profile into separate profiles 

### ISSUE THREAD

**Issue by VestboyMyst** [ripgarpr/garpr#92]
This profile is made up of several different people, all with the tag 'Seven' or '7' or 'Se7en' or some variation https://www.notgarpr.com/#/newjersey/players/56d093074d75216110485add

currently the only way to rectify this is to delete and reupload all of those tournaments, then go through and merge the proper accounts appropriately. in the future can we have a way for admins to break up an entire list of a player's events back into separate player profiles

so it would do something like 'Player X' --> 'Player X (1)' + 'Player X (2)'.... and then an admin can merge each individually / rename them accordingly

**Comment by BrandonCookeDev** [comment]
I think I could hand this easily. 

How would you like them split up tho? Considering you will be remerging some of the broken up player, we should name each resulting player differently so merging after will be simpler. 

I believe the cleanest way we could accomplish this is making each resulting player's display name read:
``` tag - tournament ```

**Comment by VestboyMyst** [comment]
yes that works, or like i said something as simple as "player (1)" , "player (2)" .... etc

**Comment by BrandonCookeDev** [comment]
I think adding the tournament to the end will avoid the hassle of double or triple checking that you chose the right number for the player. 

I guess the worst case scenario there would be rebreaking, but that might result in another set of (1) (2)...  (n)

**Comment by VestboyMyst** [comment]
alright, add on the tournament however you think would be best
